test/metabase/lib/js_test.cljs

```diff
@@ -592,8 +592,7 @@
       (are [mode expr] (-> (lib.js/diagnose-expression query 0 mode expr js/undefined)
                            .-message
                            string?)
-        "expression"  #js ["/"   #js ["field" 1 #js {:base-type "type/Address"}] 100]
-        "filter"      #js ["sum" #js ["field" 1 #js {:base-type "type/Integer"}]]))
+        "expression"  #js ["/"   #js ["field" 1 #js {:base-type "type/Address"}] 100]))
```

Comment:
@bshepherdson @ranquild 

Let me know if you think this test should be updated rather than removed, or indeed if the test is fine and this is a bug to be fixed.